### PR TITLE
CONSOLE-4234,CONSOLE-4296: Remove deprecated PF table from Cons…

### PR DESCRIPTION
**Pull request by TheRealJon**
…olePluginList

### CODE REVIEW

File: frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx [hunk 1]
```diff
@@ -96,51 +102,81 @@ const ConsolePluginStatus: React.FC<ConsolePluginStatusType> = ({ enabled, plugi
   );
 };
 
+type UseTableSort = () => [ISortBy, OnSort];
```
Comment (TheRealJon) [inline frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx:105]: TODO: Move this to console-shared

**Comment by TheRealJon** [comment]
QE Approver
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @reestr

**Comment by yanpzhan** [comment]
Regression test about console operator page(including consoleplugin page) against the pr code passed.
/label qe-approved

**Review by jhadvig — APPROVED** [review]
Nice work @TheRealJon 👍 
/lgtm

**Comment by TheRealJon** [comment]
/label px-approved

@reestr Approved above, but the label wasn't added.

**Pull request merged**